packages/tools/examples/stackManipulationToolsTouch/index.ts

```diff
@@ -0,0 +1,143 @@
+import { RenderingEngine, Types, Enums } from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  PanTool,
+  WindowLevelTool,
+  StackScrollTool,
+  ZoomTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { ViewportType } = Enums;
+const { MouseBindings } = csToolsEnums;
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Basic Stack Manipulation',
+  'Manipulation tools for a stack viewport'
+);
+
+document.body.style.overflow = 'hidden';
+document.body.style.touchAction = 'none';
+
+const content = document.getElementById('content');
+const element = document.createElement('div');
+
+// Disable right click context menu so we can have right click tools
+element.oncontextmenu = (e) => e.preventDefault();
+
+element.id = 'cornerstone-element';
+element.style.width = '100vw';
+element.style.height = '300px';
+document.body.style.margin = '0';
+
+content.appendChild(element);
+
+const instructions = document.createElement('p');
+instructions.innerText = `
+ One finger will change WW/WL,
+ Two fingers will allow for pan/zoom,
+ Three fingers will allow for stack scrolling
+`;
+
+content.append(instructions);
+// ============================= //
+
+/**
+ * Runs the demo
+ */
+async function run() {
+  // Init Cornerstone and related libraries
+  await initDemo();
+
+  const toolGroupId = 'STACK_TOOL_GROUP_ID';
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(PanTool);
+  cornerstoneTools.addTool(WindowLevelTool);
+  cornerstoneTools.addTool(StackScrollTool);
+  cornerstoneTools.addTool(ZoomTool);
+
+  // Define a tool group, which defines how mouse events map to tool commands for
+  // Any viewport using the group
+  const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
+
+  // Add tools to the tool group
+  toolGroup.addTool(WindowLevelTool.toolName);
+  toolGroup.addTool(PanTool.toolName);
+  toolGroup.addTool(ZoomTool.toolName);
+  toolGroup.addTool(StackScrollTool.toolName);
+
+  // Set the initial state of the tools, here all tools are active and bound to
+  // Different mouse inputs
+  toolGroup.setToolActive(ZoomTool.toolName, {
+    bindings: [{ numTouchPoints: 2 }],
+  });
+  toolGroup.setToolActive(StackScrollTool.toolName, {
+    bindings: [{ numTouchPoints: 3 }],
+  });
+  toolGroup.setToolActive(WindowLevelTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Left Click
+      },
+    ],
+  });
```

Comment:
Same here

Reply:
I remember you said this 

```
Yeah numTouchPoints = 1 is a special case where we also treat it as a "Primary Mouse" event. This way existing modes which just bind a primary mouse get the free touch win without additional explicit config.
```

But don't think we should treat the numPoint ===1 special, why not for the others? why just 1 ?